src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -673,17 +687,37 @@ class SpanBar extends React.Component<SpanBarProps, SpanBarState> {
   }
 }
 
-const SpanRowCellContainer = styled('div')`
-  position: relative;
-  height: ${SPAN_ROW_HEIGHT}px;
-`;
+const getBackgroundColor = ({
+  showStriping,
+  showDetail,
+}: {
+  showStriping?: boolean;
+  showDetail?: boolean;
+}) => {
+  if (showDetail) {
+    return theme.offWhite2;
```

Comment:
You should be able to use `theme` from the function argument, no?

Reply:
Yeah I think it's pretty easy—I just have to also add `theme` to the types.